data-es6.js

```diff
@@ -2914,6 +2914,30 @@ exports.tests = [
   },
 },
 {
+  name: 'prototype of bound functions',
+  category: 'functions',
+  significance: 'small',
+  link: 'https://people.mozilla.org/~jorendorff/es6-draft.html#sec-boundfunctioncreate',
+  exec: function () {/*
+    function getProtoBound(proto) {
+      var f = Object.setPrototypeOf(function() { }, proto)
+      var boundF = Function.prototype.bind.call(f, null)
+      return Object.getPrototypeOf(boundF)
+    }
+
+    return [ Function.prototype, function(){}, [], null ].every(function(proto) {
```

Comment:
I would love all tests to provide an N/A answer when the thing they depend on (as opposed to the thing itself) is simply not supported yet in the engine. An alternative here would be for the subtest to pass - because arrow functions that aren't present don't fail, for example.